x/rewards/keeper/distribution.go

```diff
@@ -81,7 +79,7 @@ func (k Keeper) estimateBlockGasUsage(ctx sdk.Context, height int64) *blockRewar
 					TxGasUsed:           make(map[uint64]uint64, 0),
 					InflationaryRewards: sdk.Coin{Amount: sdk.ZeroInt()}, // necessary to avoid nil pointer panic on Coins.Add call
 				}
-				if metadata, found := metadataState.GetContractMetadata(contractDistrState.ContractAddress); found {
+				if metadata, err := k.ContractMetadata.Get(ctx, contractDistrState.ContractAddress); err == nil {
```

Comment:
Is it safe to ignore errors in this context?
What is the intent if there are errors?